FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/BaseHealthChecker.cs

```diff
@@ -44,15 +48,19 @@ public async Task<HealthCheckResult> PerformHealthCheckAsync(CancellationToken c
                     Status = HealthCheckStatus.UNHEALTHY,
                     ErrorMessage = $"Unhandled exception : {e.Message}.",
                 };
-                _logger.LogError($"Unhandled exception occured in health check component {Name}. {e.Message}");
+
+                _diagnosticLogger.LogWarning($"Unhandled exception occured in health check component {Name}. {e.Message}");
+                _logger.LogInformation($"Unhandled exception occured in health check component {Name}. {e.Message}");
             }
 
             if (healthCheckResult.Status is HealthCheckStatus.UNHEALTHY)
             {
+                _diagnosticLogger.LogWarning($"Health check component {Name} is unhealthy. Failed reason: {healthCheckResult.ErrorMessage}");
```

Comment:
The health check item might contain some information about internal implementations, like the job table and queue, will these messages also be passed into diagnostic logger?

Reply:
It will not contain internal message (table or queue).
The only message for internal resource will look like: "Scheduler service is inactive."